docs/spec/spec.md

```diff
@@ -76,15 +76,15 @@ _Describes a linkage between a Channel and a Targetable and/or Sinkable._
 
 #### Spec
 
-| Field              | Type           | Description                                                                  | Constraints        |
-| ------------------ | -------------- | ---------------------------------------------------------------------------- | ------------------ |
-| from\*             | ObjectRef      | The originating _Subscribable_ for the link.                                 | Must be a Channel. |
-| call<sup>1</sup>   | EndpointSpec   | Optional processing on the event. The result of call will be sent to result. |                    |
-| result<sup>1</sup> | ResultStrategy | The continuation for the link.                                               |                    |
+| Field                  | Type           | Description                                                                         | Constraints        |
+| ---------------------- | -------------- | ----------------------------------------------------------------------------------- | ------------------ |
+| channel\*              | ObjectRef      | The originating _Subscribable_ for the link.                                        | Must be a Channel. |
+| subscriber<sup>1</sup> | SubscriberSpec | Optional processing on the event. The result of subscriber will be sent to replyTo. |                    |
+| replyTo<sup>1</sup>    | ReplyStrategy  | The continuation for the link.                                                      |                    |
```

Comment:
I'm still not a fan of `replyTo` being a verb+preposition. What about `result` as the name for this field so that we have three nouns?

Reply:
We could change it to replyChannel. I don't like adding another term ("result") unless we have a good reason. Typically messaging systems would refer to this as "reply", and when we do add support for dynamic resolution of the reply Channel, we would likely use a "replyTo" header. That could end up being confusing if this property name is the same, so I'm all for changing it.

Reply:
I would only support `replyChannel` if we also flatten the current ResultStrategy object.

The yaml with the current content of this PR looks like:
```yaml
...
spec
  channel: ...
  replyTo:
    channel:
      kind: Channel
      apiGroup: eventing.knative.dev/v1alpha1
      name: my-channel-2
```

With rename and flattening
```yaml
...
spec
  channel: ...
  replyChannel:
    kind: Channel
    apiGroup: eventing.knative.dev/v1alpha1
    name: my-channel-2
```